etl-pipeline/managers/cover_fetchers/content_cafe_fetcher.py

```diff
@@ -4,12 +4,12 @@
 from requests.exceptions import ReadTimeout
 
 from logger import create_log
-from managers.coverFetchers.abstractFetcher import AbstractFetcher
+from managers.cover_fetchers.fetcher_abc import FetcherABC
 
 logger = create_log(__name__)
 
 
-class ContentCafeFetcher(AbstractFetcher):
+class ContentCafeFetcher(FetcherABC):
```

Comment:
still learning python but is it common to suffice abstract ABC classes with ABC?

Reply:
That is the naming convention per the [docs](https://docs.python.org/3/library/abc.html)